OrthoEvol/Manager/database_management.py

```diff
@@ -1,682 +1,808 @@
+            taxdump.write(data)
+        with tarfile.open(str(dl_abs_path)) as tar:
+            tar.extractall(dl_path)
+        os.remove(dl_abs_path)
+
+    def download_refseq_release_files(self, collection_subset, seqtype, seqformat):
+        """
+        Download NCBI Refseq Release files from NCBI.  The collection subtype is a species group
+        (e.g. vertebrate_mammalian) of interest.
+
+        :param collection_subset: The collection subset of interest.
+        :type collection_subset: str.
+        :param seqtype: The type of sequence (RNA, protein, genomic).
+        :type seqtype: str.
+        :param seqformat: The format of the sequence file (usually 'gbff' for GenBank Flat File).
+        :type seqformat: str.
+        :return: A list of files to download from NCBI via FTP.
+        :rtype:  list.
+        """
+        db_path = self.database_path / Path('NCBI') / Path('refseq') / Path('release') / Path(collection_subset)
+        db_path.mkdir(parents=True, exist_ok=True)
+        # TODO: If database exists and is same size, use the existing database.
+        self.ncbiftp.getrefseqrelease(collection_subset=collection_subset, seqtype=seqtype, 
+                                      seqformat=seqformat, download_path=db_path)
+        return self.ncbiftp.files2download
+
+    def upload_refseq_release_files(self, collection_subset, seqtype, seqformat, upload_list=None, database_name=None, add_to_default=None):
+        """
+        Upload NCBI's Refseq Release files to a BioSQL database.
+
+        :param collection_subset: The collection subset of interest.
+        :type collection_subset: str.
+        :param seqtype: The type of sequence (RNA, protein, genomic).
+        :type seqtype: str.
+        :param seqformat: The format of the sequence file (usually 'gbff' for GenBank Flat File).
+        :type seqformat: str.
+        :param upload_list:  A list of files to upload.
+        :type upload_list:  list.
+        :param database_name:  The name of the database to create.  The default name is usually best.
+        :type database_name:  str.
+        :param add_to_default:  A string to add to the default name.
+        :type add_to_default:  str.
+        """
+
+        if database_name:
+            db_name = database_name
+        else:
+            if add_to_default:
+                add_to_default = "_%s" % add_to_default
+            else:
+                add_to_default = ""
+            db_name = "{}_{}{}.{}.db".format(collection_subset, seqtype, add_to_default, seqformat)
+        db_path = self.database_path / Path("NCBI") / Path("refseq") / Path("release") / Path(collection_subset)
+        # Get a BioSQL database
+        ncbi_db = self.copy_biosql_taxonomy_template(destination=db_path, database_name=db_name)
+        ncbi_db.upload_files(seqtype=seqtype, filetype='genbank', upload_path=db_path, upload_list=upload_list)
+
+    def get_project_genbank_database(self):
+        """"""
+        print()
```

Comment:
Add a pass here if you want. Not necessary.

Reply:
```suggestion
        pass
```